sql/updates/latest-dev.sql

```diff
@@ -0,0 +1,4 @@
+
+-- set compressed_chunk_id to NULL for dropped chunks
+UPDATE _timescaledb_catalog.chunk SET compressed_chunk_id = NULL WHERE dropped = true AND compressed_chunk_id IS NOT NULL;
```

Comment:
Won't this orphan a compressed chunk if the uncompressed chunk is marked as dropped? In short, what if `compressed_chunk_id` refers to a chunk that actually exists?

Reply:
no we only keep metadata of uncompressed chunk